src/SyntaxHighlighter/Cpp.cpp

```diff
@@ -97,6 +97,62 @@ Cpp::Cpp() : QSyntaxHighlighter{static_cast<QObject*>(nullptr)} {
   class_member_regex_ = QRegularExpression{CppRegex::kClassMemberRegex};
 }
 
+CppHighlighterState HighlightBlockCpp(
```

Comment:
This function needs also be declared in the header file. Otherwise you can't call it from somewhere else.

Reply:
Yes, of course. I only wanted to receive comments about how to implement it, sorry for that.